### ISSUE TITLE
Creating multiple MobileScanners in an app does not work

### ISSUE THREAD

**Issue by Luzzotica** [juliansteenbakker/mobile_scanner#196]
I need to have multiple QR Code scanners in my app. 
My attempts to get this working have been rather fruitless:
1. Trying to turn on/off the mobile scanners as I move beteween screens
2. Having a global mobile scanner that I insert into the screens that need it.

I'm going to keep working at it, next attempt will use option number 2, but embed it into a page I can slide up from the bottom at will.

Are there any tips or tricks you would recommend for using a QR code scanner in multiple locations?
Any help would be appreciated, thank you!

- Sterling Long!

**Comment by fasterlars** [comment]
Hi. 
I´m having the same problem. Did you find a solution? :)

**Comment by fasterlars** [comment]
I tried creating a Singleton-sort-of instance of the scanner. I also tried to do whatever possible to dispose and null the scanner when the view disposed. With timers and what-not. But no dice. 
I often get the `Called start() while already starting` and `PlatformException(error, No active stream to cancel, null, null)`. At what seems to be unpredictable times. 
Will continue to see if I can find a solution.
